openquake/risklib/asset.py

```diff
@@ -142,8 +142,8 @@ def get_units(self, loss_types):
                 lt = lt[:-4]
             elif lt == 'number':
                 unit = 'units'
-            elif lt == 'area':
-                unit = 'unknown'
+            elif lt == 'area' and lt not in self.units:
+                unit = 'SQM'
```

Comment:
Is it actually possible to have 'area' inside `self.units`? If yes, add a comment saying in which test this happens. If not, the `and` part is useless.